app/controllers/reports_controller.rb

```diff
@@ -122,6 +136,14 @@ def add_data_to_sheet_for_export_applications(sheet, applications)
     end
   end
 
+  def getCountsForUserSignInReport(one_week_where_clause, two_weeks_where_clause, under_a_month_where_clause, over_a_month_where_clause)
```

Comment:
@SeanLF generally all method names would use the underscore case rather than camel case. So

     get_counts_for_users_sign_in_report